### PR TITLE
Fix [BUG] Module 'PIL' has no attribute Image

### PR DESCRIPTION
**Pull request by Future-Outlier**
# TL;DR
_Please replace this text with a description of what this PR accomplishes._

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
You can search "return PIL.Image.open(local_path)" in the code, and you will find that it is not enough to just use "import PIL" in the code.

## Tracking Issue
https://github.com/flyteorg/flyte/issues/3813

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: plugins/flytekit-deck-standard/flytekitplugins/deck/renderer.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 from typing import TYPE_CHECKING, List, Optional, Union
 
+import PIL.Image
```
Comment (kumare3) [inline plugins/flytekit-deck-standard/flytekitplugins/deck/renderer.py:3]: I do not think we need this? We just use PIL.Image.Image so line 17 should be enough right?

**Review by kumare3 — CHANGES_REQUESTED** [review]
I do not think we should add the import, there is a lazy load already

**Comment by Future-Outlier** [comment]
> I do not think we should add the import, there is a lazy load already

Just found that you are right, thanks a lot for the review!

**Comment by Future-Outlier** [comment]
Just contact the author of this pull request, I think maybe the bug doesn't exist.
https://github.com/flyteorg/flytekit/blob/master/plugins/flytekit-deck-standard/tests/test_renderer.py#L60-L72
Our test has covered the case he've mentioned.

File: plugins/flytekit-deck-standard/flytekitplugins/deck/renderer.py (hunk 1 above)
Comment (Future-Outlier) [reply plugins/flytekit-deck-standard/flytekitplugins/deck/renderer.py:3]: Yes, you are probably right.

**Pull request closed without merging**